MetaMask/metamask-mobile
# refactor: [Part 1] - Ongoing Work for legacy <Text> comp replacement.

## **Description**
This PR replaces old usage of legacy <Text> component with the text component that standardizes on typography provided through theme via [@metamask/design-tokens](https://www.npmjs.com/package/@metamask/design-tokens) library. This will bring uniformity to all text elements on our app.

## **Manual testing steps**
- From the main screen, click on any asset available to go to Asset Detail.
- Make sure  you have no balance on the chosen asset [Specific for testing Warning Message Text].
- Review Button's Texts.

## **Screenshots/Recordings**

### **Before**
![Screenshot 2023-10-20 at 14 54 13](https://github.com/MetaMask/metamask-mobile/assets/44679989/75ab745a-1539-43fe-890e-dfc24145e42d)
![Simulator Screen Shot - iPhone 12 Pro - 2023-10-23 at 12 17 49](https://github.com/MetaMask/metamask-mobile/assets/44679989/46085891-262f-42b9-8b65-2eb42b7489c8)


### **After**
![Screenshot 2023-10-20 at 15 07 52](https://github.com/MetaMask/metamask-mobile/assets/44679989/da4bf332-f009-49fc-aa1c-cccf921aede1)
![Simulator Screen Shot - iPhone 12 Pro - 2023-10-23 at 12 27 21](https://github.com/MetaMask/metamask-mobile/assets/44679989/daae33d8-b229-4cc6-bd05-36d989567283)

## **Related issues**

This is **part** of ongoing tasks related to issue #6887

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
